src/libraries/sendtohelixhelp.proj

```diff
@@ -98,11 +99,19 @@
     <HelixPreCommand Include="export XHARNESS_LOG_WITH_TIMESTAMPS=true" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(Scenario)' == 'WasmTestOnBrowser'">
+  <ItemGroup Condition="'$(Scenario)' == 'WasmTestOnBrowser' or '$(Scenario)' == 'BuildWasmApps'">
     <HelixPreCommand Include="export PATH=$HELIX_CORRELATION_PAYLOAD/chromedriver_linux64:$PATH" />
     <HelixPreCommand Include="export PATH=$HELIX_CORRELATION_PAYLOAD/chrome-linux:$PATH" />
   </ItemGroup>
 
+  <PropertyGroup Condition="'$(Scenario)' == 'BuildWasmApps'">
+    <IncludeXHarnessCli>true</IncludeXHarnessCli>
```

Comment:
Looks like `IncludeXHarnessCli` is already defined on L73 under `'$(TargetOS)' == 'Browser'`.